lib/hg-utils.coffee

```diff
@@ -291,7 +292,10 @@ class Repository
         if err
           reject err
         if stderr?.length > 0
-          reject stderr
+          if isWindows and stderr.length > windowsMaxPathLength
+            resolve(stdout)
```

Comment:
I don't think that works since the error message could be localized, so if I had the dutch language pack enabled it would be 'bestand kon niet gevonden worden' or somesuch.

...there's got to be a better way though.